dev/tools/controllerbuilder/generate-proto.sh

```diff
@@ -67,12 +67,15 @@ protoc --include_imports --include_source_info \
     ${REPO_ROOT}/mockgcp/apis/mockgcp/cloud/apigee/*/*.proto \
     ${REPO_ROOT}/mockgcp/apis/mockgcp/cloud/cloudidentity/groups/*/*.proto \
     ${REPO_ROOT}/mockgcp/apis/mockgcp/cloud/networkconnectivity/*/*.proto \
+    ${THIRD_PARTY}/googleapis/google/*/*.proto \
+    ${THIRD_PARTY}/googleapis/google/api/*.proto \
```

Comment:
```
${THIRD_PARTY}/googleapis/google/api/*.proto \
```
is both a duplicated line and not needed as it it already be covered by the following
```
${THIRD_PARTY}/googleapis/google/*/*.proto \
```